crates/load-cargo/src/lib.rs

```diff
@@ -265,6 +265,11 @@ impl ProjectFolders {
                 entries.push(manifest.to_owned());
             }
 
+            for buildfile in ws.buildfiles() {
+                file_set_roots.push(VfsPath::from(buildfile.to_owned()));
+                entries.push(buildfile.to_owned());
```

Comment:
Would be nice to avoid these `.to_owned()` calls if possible.

Reply:
`VfsPath` owns the `AbsPathBuf`, so I'm not sure if it can be avoided. fwiw, this is copied from above: https://github.com/rust-lang/rust-analyzer/blob/85ca217765a61db7d72783a18f77db6ef7022637/crates/load-cargo/src/lib.rs#L263-L266